### ISSUE TITLE
using Summary description instead of Event description in tooltip popup 

### ISSUE THREAD

**Issue by FrancSimari** [DNNCommunity/DNN.Events#111]
## Is your feature request related to a problem?
**Please describe.**
Tooltip pop up allows default 10,000 characters and could be a big popup making it impossible to click in the event
## Describe the solution you'd like
Use the summary description instead of using the event description.

**Comment by valadas** [comment]
This should be a setting in order to not change behaviour on existing users who expect the description to be in the popup.

**Comment by FrancSimari** [comment]
It seems to me it is the actual event that popup not the description and I don't see a setting for it. The only way I can control the popup amount of content is by cutting it from 10000 to 200 characters. 
Thanks for you input, I'll revisit the setting just in case I missed something.

**Comment by FrancSimari** [comment]
If I can step back in, I think allowing the user to select is good, my preference would be no option at all, the summary will be used for the popup and you can also limit the characters to e.g. 250 the full description will display after the link is clicked.  Less required steps is always better. The summary is more steps for the user, for less steps you can allow just the "first 250 characters plus ...more" of the description to display. With the latter you won't need the summary option or the limit on the description.

**Comment by FrancSimari** [comment]
Sorry didn't mean to close it maybe someone else want to contribute

**Comment by valadas** [comment]
I disagree because of upgrades, this would change the current behaviour on users that already have it working the way they want. I think this should be either a setting or included in a new template (I have to check on templates on this module first), but a solution that would not change the popup content without a user action first in order to not change the behaviour on upgrades automatically.

**Comment by FrancSimari** [comment]
I will have to agree with you. I don't know all the history or dependencies.

**Comment by EPTamminga** [comment]
There is a template for the tooltip. 
If people want the summary in the tooltip, it is just a matter of changing template. 
No need to change anything or adding a setting.

**Comment by valadas** [comment]
So basically, these two links should document how to modify a template:

https://github.com/DNNCommunity/DNN.Events/wiki/The-use-of-Templates
https://github.com/DNNCommunity/DNN.Events/wiki/Tokens-to-be-used-in-Templates

Changing the token used in the tooltip template should resolve the issue for this situation without changing anything in the code.

**Comment by EPTamminga** [comment]
I will give it a try later this week. Stay tuned.

**Comment by thabaum** [comment]
This is a very old issue I was surprised to not see my two cents on this as I know I posted about it a few times... and is annoying

https://github.com/DNNCommunity/DNN.Events/issues/93

This post has my 2 cents on it.

Also the popup is very useful in a way but gets in the way... I seen a magnify glass on one of my listings when I set there to be a custom page link for the event.

I could not find the custom page link anywhere so I will make another issue now on that.

I believe that popup "Summary" should only show up when you hover over that magnify glass.  And when you click the link for the event it shows up with "Full Description" it should show a link to that custom page.
